alecs-theme/src/WEB-INF/src/resources-importer/journal/templates/Service/Service.ftl

```diff
@@ -0,0 +1,35 @@
+<style>
+    .service-row {
+        background: ${BackgroundColor.getData()};
+        border-radius: 15px;
+        color: #FFF;
+        margin: 0px 15px;
+        padding: 20px;
+        text-align: center;
+    }
```

Comment:
Is there anyway to integrate the CSS into the theme and just allow what needs to be customized to be here in the Freemarker template?